### DISCUSSION TITLE
txt file on github not being fetched due to CORS policy despite web app hosted on github

### DISCUSSION THREAD

**Discussion by yashineonline** [community/community#136949] [New to GitHub]
txt file on github not being fetched due to CORS policy despite web app hosted on github
can anyone help please?

**Comment by ghostinhershell** [comment]
Hi @yashineonline, we're happy you're here! You are more likely to get a useful response if you are more explicit about the issue you're running into here, giving a few more details might help someone give you a nudge in the right direction.

**Reply by yashineonline** [reply]
thank you for the response. the issue is  solved. you may delete this post maybe?
